src/jasmineTestRunner/JasmineReporter.js

```diff
@@ -185,54 +149,100 @@ function _extractSpecResults(container, ancestorTitles, spec) {
           'Unexpected jasmine spec result type: ', result.type
         );
     }
-  });
+  }, this);
 
   container.push(results);
-}
-
-function JasmineReporter() {
-  jasmine.Reporter.call(this);
-  this._logs = [];
-  this._resultsDeferred = Q.defer();
-}
-JasmineReporter.prototype = Object.create(jasmine.Reporter.prototype);
+};
 
-// All describe() suites have finished
-JasmineReporter.prototype.reportRunnerResults = function(runner) {
-  var testResults = [];
+JasmineReporter.prototype._highlightDifferences =
+function (a, b) {
```

Comment:
Nah, seems it fits 80 cols, will put on the same (otherwise, that's pattern is used).